### PR TITLE
Comment on old bugs with new failed test url

### PR DESCRIPTION
**Pull request by asdil12**

### CODE REVIEW

File: openqa_review/openqa_review.py [hunk 1]
```diff
@@ -177,6 +177,13 @@ def pluralize(_1, _2, plural):
 **Status: $status_badge**
 $new_product_issues$existing_product_issues$new_openqa_issues$existing_openqa_issues$todo_issues""")
 
+openqa_issue_comment = Template("""
+This is an autogenerated message for openQA integration by the openqa_review script:
+
+This bug is still assigned to a failing openQA test module: $name
```
Comment (okurz) [inline openqa_review/openqa_review.py]: Better say "This bug is still referenced in a failing openQA test"

File: openqa_review/openqa_review.py [hunk 2]
```diff
@@ -1001,12 +1050,39 @@ def filter_report(report, iefilter):
     report.report = SortedDict({p: pr for p, pr in iteritems(report.report) if pr.reports})
 
 
+def reminder_comment_on_issue(ie):
+    issue = ie.bug
+    if (datetime.datetime.utcnow() - issue.last_comment).days >= 14:
```
Comment (okurz) [inline openqa_review/openqa_review.py]: extract the magic number as a configurable function parameter with 14 as default, please

**Comment by asdil12** [comment]
It is actually done but missing tests (coverage is 93%).
Also note that the existing config will not work with the new version.
You will need to remove the `%(username)s:%(password)s@` from the bugzilla `base_url` and add the `api_key` entry for redmine.

**Comment by okurz** [comment]
coverage incomplete on these lines: 915, 914->915, 916->918, 1079->1075

please check output of `tox -e cov`

**Pull request merged**